src/autolabel/confidence.py

```diff
@@ -255,7 +256,7 @@ async def _call_with_retry(self, model_input, model_output):
         headers = {"refuel_api_key": self.REFUEL_API_KEY}
         async with httpx.AsyncClient() as client:
             response = await client.post(
-                self.BASE_API, json=payload, headers=headers, timeout=30
+                self.endpoint, json=payload, headers=headers, timeout=30
```

Comment:
let's check and handle the case where this isn't passed ? we can return a score of 0.0 + log an error